test/release/examples/PREDIFF

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
-sort $2 > $2.prediff.tmp
+sed -e 's/\r$//' -e '/^salloc:/d' < $2 | sort > $2.prediff.tmp
```

Comment:
We used to have a global prediff to filter out stuff that `expect` introduced (the carriage return and salloc commands). It was removed with 1c603a939aba2ca8a98e6e11efa3af7c919eaed4, but you can probably just resurrect that and use it. Longer term we probably want to update slurm-gasnetrun_ibv to be more like slurm-srun (or really find a way to merge them into one so we don't have to duplicate code) but i think a global prediff is good enough to get started